### ISSUE TITLE
Images missing max-width property 

### ISSUE THREAD

**Issue by paskal** [yoshiharuyamashita/blackburn#68]
Hello!

I've stumbled upon the issue exactly the same as https://github.com/dim0627/hugo_theme_beg/issues/19:
![image](https://user-images.githubusercontent.com/712534/37654054-ecc1413a-2c51-11e8-89c8-1303ce99e220.png)

Source code:

    ![Shinken simple distributed architecture](https://shinken.readthedocs.io/en/latest/_images/shinken-architecture.png)

**Comment by yoshiharuyamashita** [comment]
Hi @paskal 

Does wrapping your image using `fluid_imgs` fix the problem? Take a look at [this](http://yoshiharuyamashita.com/post/hugo-shortcode-to-show-multiple-images/) for usage.

**Comment by paskal** [comment]
It could fix the problem but it won't change the fact standard `![image_alt_text](image_link)` is rendered improperly.

**Comment by nonumeros** [comment]
@paskal the only md engine I'm aware of (there may be others out there though) is https://github.com/gettalong/kramdown which would let you have `![alt name](image link){:width=dim height=dim}` 
You should ask the hugo devs

See https://github.com/takuti/takuti.me/blob/master/_content/note/hugo-markdown-and-mathjax.md

**Comment by paskal** [comment]
Guys, guys, it's as simple as https://github.com/dim0627/hugo_theme_beg/commit/7718359506a2447ef0489dcd6bd7360f21eb93cb

**Comment by nonumeros** [comment]
> Guys, guys, it's as simple as dim0627/hugo_theme_beg@7718359

You're right.... for `max-width` you have a `pure-img-responsive` on fluids_images.html.... So replacing the `.pure-img-responsive` on side-menu.css seems to resolve it.

**Comment by yoshiharuyamashita** [comment]
Perhaps would it not be better to create your own custom css with the fix and use it with `custom_css`?

**Comment by paskal** [comment]
Create my own custom_css to get generic markdown syntax properly rendered? I've got you.

**Comment by nonumeros** [comment]
@paskal thanks for bringing it up. If you were to modify ( as per your suggestion with https://github.com/dim0627/hugo_theme_beg/commit/7718359 ), **side-menu.css** 's `pure-img-responsive` with something as simple as `img` , further changes on this theme are still required. 

Try it out... It would render momentarily.

I think other hugo themes may benefit from your finding. I'll reference you accordingly.
